sshuttle/client.py

```diff
@@ -45,6 +45,8 @@ def got_signal(signum, frame):
     log('exiting on signal %d\n' % signum)
     sys.exit(1)
 
+ALWAYS_CONNECTED_ON = "ON"
```

Comment:
Can you make this boolean True and the other boolean False?   It doesn't need to be text and will be more efficient.